### DISCUSSION TITLE
[Feature] Flexible and optional family sharing

### DISCUSSION THREAD

**Discussion by GITHUBNULLING** [immich-app/immich#14010] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Everyone has privacy, and sharing of family albums is inevitable. I know that immich provides some ways to let family members see what they should see, but it is not flexible and convenient enough. For example, some photos in a certain album are not suitable for moving to other albums, but these photos are not suitable for showing to family members, and I don’t want to remove these photos, so there needs to be an exclusion function. In another case, there may be some folder contents in the source folder of the external gallery that are not suitable for family sharing. If some folders can be excluded and can still exist in my own album and my timeline, it would be perfect. I hope to add the following functions: **1. Exclusion of single photos. 2. Exclusion of folders of source files.** I believe immich will become more and more perfect.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4500/#6350/#2654
